taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -64,6 +65,9 @@ const StartProject = ({ clearNavBarLinks }) => {
     setErrorMessage("");
   };
 
+
+  const textFieldError = BranchNameRegEx.test(branchName);
```

Comment:
think this will be `true` if is the branchName is valid.  Please rename as `isValidBranchName`

Reply:
@nobeeakon I have changed the name. Thanks!

Reply:
looks great, just please update the places where this variable is used, like disabled and helper text, and error, to `!isValidBranchName`

Reply:
Actually the above statement returns true if there exists a special character(invalid name). It returns false if branch name is valid. So should I just change the name to `isInvalidBranchName`

Reply:
yes please use `isInvalidBranchName`, also please use `branchNameRegEx` instead of `BranchNameRegEx`  . Sorry for the confusion.